setup.cfg

```
@@ -1,6 +1,6 @@
 [metadata]
 name=py-seed
-version=0.4.3
+version=0.4.4
```

I think we should bump to 0.5 due to the breaking change to previous versions. 

```suggestion
version=0.5.0
```